astro/src/styles/bootstrap.scss

```diff
@@ -360,10 +381,14 @@ svg[data-icon="loca11y-logo"].unpad {
 }
 
 @include color-mode(dark) {
-  *:focus-visible, .form-field:focus, .navbar-toggler:focus, .btn:focus-visible, .nav-link:focus-visible {
+  *:focus-visible, .form-field:focus, .form-check-input:focus, .navbar-toggler:focus, .btn:focus-visible, .nav-link:focus-visible {
     outline-color: #FFFFFF;
   }
 
+  .form-field:focus {
+    border-color: $info-border-subtle-dark;
```

Comment:
Again, this only works if we make all of the forms the info theme. Any other ideas?